### PR TITLE
Added TFLite Model Prediction Script with Image Preprocessing

### PR DESCRIPTION
**Pull request by rishimj**
### **Summary:**
- Introduces a Python script for image classification using a TensorFlow Lite (TFLite) model.
- Key features include:
  - **Image Preprocessing:** Handles base64 decoding, resizing, normalization, and formatting for TFLite models.
  - **Model Prediction:** Runs inference, applies `softmax`, and outputs the predicted class with confidence scores.
  - **Error Handling:** Comprehensive handling for model loading, input processing, and prediction issues.
- Accepts JSON input via `stdin` with a base64-encoded image and MIME type.
- Outputs JSON with `predicted_class` and `confidence`.
- Configures logging at the `INFO` level for better debugging.
- Example output:
  ```json
  {
    "predicted_class": "mediumCherry",
    "confidence": 98.45
  }

### CODE REVIEW

**Review by Czh0630 — COMMENTED** [review]
good
